.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 nbproject
 bower_components
+dist/*
```

Comment:
This is required for the bower packaging so unfortunately we can't explicitly ignore this right now.